source/crud/transactions.txt

```diff
@@ -304,19 +304,39 @@ the ``startTransaction()`` method:
 Transaction Errors
 ------------------
 
-If you are using the Core API to perform a transaction, you must incorporate
-error-handling logic into your application for the following errors:
-
-- ``TransientTransactionError``: Raised if a write operation errors
-  before the driver commits the transaction. To learn more about this error, see the
-  :manual:`TransientTransactionError description
-  </core/transactions-in-applications/#transienttransactionerror>` on
-  the Driver API page in the Server manual.
-- ``UnknownTransactionCommitResult``: Raised if the commit operation
-  encounters an error. To learn more about this error, see the
-  :manual:`UnknownTransactionCommitResult description
-  </core/transactions-in-applications/#unknowntransactioncommitresult>` on
-  the Driver API page in the Server manual.
-
-The Convenient Transaction API incorporates retry logic for these error
-types, so the driver retries the transaction until there is a successful commit.
\ No newline at end of file
+Since MongoDB transactions are :website:`ACID compliant
```

Comment:
S: Add a Convenient Transaction API subheading to distinguish from Core API, similar to the Transaction APIs section earlier on the page. The list of errors can go before the subheadings.